AirCasting/SearchAndFollow/SearchLocation/SearchViewModel.swift

```diff
@@ -29,7 +31,8 @@ class SearchViewModel: ObservableObject {
         return sensorType(from: selected.name)
     }
     
-    init() {
+    init(isSearchAndFollowFlowActive: Binding<Bool>) {
+        self.isSearchAndFollowFlowActive = isSearchAndFollowFlowActive
```

Comment:
I am not sure about this one, because then you need to import the SwiftUI to the VM which is quite not the concept that we are following. Using the SwiftUI in the AC app, we needed to use this 'hack' for dismissing, but it is totally a UI thing. Don't you think it should stay on the UI layer? Don't have strong opinion as I am not the master 😟